access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java

```diff
@@ -1110,21 +1108,20 @@ public long getCollectionsCount() {
 	 * @return a map from format name to count
 	 */
 
-	public Map<String, Long> getFormatCounts() {
-
-		SolrQuery query;
-		QueryResponse response;
-
-		query = new SolrQuery();
-		query.setQuery("*:*");
-		query.setRows(0);
-		query.addFacetField("contentType");
-		query.setFacetLimit(-1);
-
+	public Map<String, Long> getFormatCounts(AccessGroupSet accessGroups) {
 		HashMap<String, Long> counts = new HashMap<String, Long>();
```

Comment:
are we not using the newer syntax for declaring collections variables?

HashMap<String, Long> counts = new HashMap<>();